### ISSUE TITLE
evidence code decision

### ISSUE THREAD

**Issue by ValWood** [pombase/website#109]
Do we want to show GO evidence code in full?
It's nice to see it, but it is now quite prominent.

Thoughts?  Ideas? No hurry for this decision.....

**Comment by kimrutherford** [comment]
Currently it's showing what's in Chado.

If we decide to display the abbreviations I think it should show the full evidence code when you mouse-over.

**Comment by mah11** [comment]
I'm not hugely fussy about this, but abbreviation with full-in-mouseover sounds good for now. Later we'll have to decide what to do when it's ECO proper behind the scenes ...

**Comment by kimrutherford** [comment]
We don't have abbreviated evidence codes for FYPO.  Is it OK to be inconsistent with this?

**Comment by ValWood** [comment]
well we are currently. I'm OK with it....

**Comment by kimrutherford** [comment]
> I think it should show the full evidence code when you mouse-over.

Actually I didn't look close enough.  Currently the evidence codes are links to http://www.geneontology.org/page/guide-go-evidence-codes

I'll go with that unless I hear from you.

**Comment by kimrutherford** [comment]
For GO we now show the short evidence codes and link the docs.

**Comment by ValWood** [comment]
IEA needs to go to
http://www.geneontology.org/page/automatically-assigned-evidence-codes

**Comment by kimrutherford** [comment]
Thanks.  I spotted a couple of others that didn't go to the right place and fixed those too.  I should have checked them.

**Comment by ValWood** [comment]
hmm, they are a bit random...

**Comment by kimrutherford** [comment]
I noticed a spelling difference between GO and the Chado config file too for IBD.  Luckily you don't use it.

In our config files we spell "descendant" with an "ant" at the end.  But GO have an "ent" at the end in the URL and in the text on this page:
  http://geneontology.org/page/ibd-inferred-biological-aspect-descendent

But in the main list it has "ant" at the end:
  http://www.geneontology.org/page/guide-go-evidence-codes
"Inferred from Biological aspect of Descendant (IBD)"

Maybe I copied from the list page to get the Canto and Chado config.
